src/components/PersonLink.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams, Link } from 'react-router-dom';
+import { Person } from '../types';
+
+const findParent = (
+  person: Person,
+  parentType: string,
+  name: string,
+  peopleList: Person[],
+) => {
+  const parent = peopleList
+    .find(personToFind => personToFind.name === name);
+
+  if (parent) {
+    return (
+      <Link
+        className={classNames(
+          { 'has-text-danger': parentType === 'mother' },
```

Comment:
It would be cool not to use magic values and put such strings into constants and reuse them:
```
const MOTHER = 'mother';
const FATHER = 'father';
const NOT_SET_VALUE = '-';
```
This will increase the readability of the code and save us from possible errors.